src/js/polyfills/blob.js

```diff
@@ -242,3 +247,5 @@ Object.defineProperty(window, 'Blob', {
     writable: true,
     value: Blob
 });
+
+export { kBlobGetParts };
```

Comment:
Since this file changes the global already, importing it at the wrong time might have implications. Thus I think it's best to use a global symbol. `Symbol.for('tjs.internal.blob.getParts')`.

If you use that on both places it will return the same symbol instance, so no imports are necessary.

Reply:
You don't need to put them anywhere. Wherever you need it, just use `Symbol.for('tjs.internal.blob.getParts')` and you'll get back the same reference.